api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -263,6 +263,9 @@ message ExternalProcessor {
   // backend stream lifetime. In this case, Envoy will eventually timeout the external processor stream according to this time limit.
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
+
+  // Enable handling of :ref:`more_chunks <envoy_v3_api_field_service.ext_proc.v3.BodyMutation.more_chunks>`.
+  bool enable_more_chunks = 20;
```

Comment:
Just a quick comment: 
I noticed that here is changing from `int max_chunks` to `bool`. It seems that ext_proc server can send back any chunks as long as this it is true.

Basically, it changed from M to unlimited ?

Reply:
The side stream server can stream back many chunks. And those data are not buffered in Envoy ext_proc filter. They are directly injected into the filter and send to backend upstream: https://github.com/envoyproxy/envoy/blob/3f6a8c0f95a014550d1d8df5c849df36f9078b96/source/extensions/filters/http/ext_proc/processor_state.cc#L276.

In case of slow backend upstream and fast side stream server case, the same flow control mechanism used in the case of slow backend upstream and fast downstream client will help protect here as well. 


CC @yanavlasov

Reply:
> In case of slow upstream and fast side stream case, the same flow control mechanism used today in the case of slow upstream and fast downstream will help protect here as well.
> 
> CC @yanavlasov

flow control between upstream and downstream can NOT help here when it is unlimited sidestream injection. It could indirectly help with existing 1:1 mode though

Reply:
In the response direction, it's basically the side stream server streaming data back to Envoy then to upstream server.  The ext_proc filter is not buffering any data at all in this direction.  I hope the generic flow control mechanism will kick in here. 

How does the internal buffer in the request direction make any difference in terms of flow control on the response direction?

Reply:
oops Missed this comment. I think i have explained offline before though. Just add here for reference.

-> How does the internal buffer in the request direction(ext_proc filter->side_stream_server) make any difference in terms of flow control on the response direction?

This is misunderstanding of my comments. internal buffer and flow control on response path are two separate issues that I comment in the design doc.